extractor/src/main/java/org/schabi/newpipe/extractor/services/bandcamp/extractors/BandcampExtractorHelper.java

```diff
@@ -0,0 +1,161 @@
+// Created by Fynn Godau 2019, licensed GNU GPL version 3 or later
+
+package org.schabi.newpipe.extractor.services.bandcamp.extractors;
+
+import com.grack.nanojson.JsonObject;
+import com.grack.nanojson.JsonParser;
+import com.grack.nanojson.JsonParserException;
+import com.grack.nanojson.JsonWriter;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.schabi.newpipe.extractor.NewPipe;
+import org.schabi.newpipe.extractor.exceptions.ParsingException;
+import org.schabi.newpipe.extractor.exceptions.ReCaptchaException;
+import org.schabi.newpipe.extractor.localization.DateWrapper;
+import org.schabi.newpipe.extractor.utils.Utils;
+
+import java.io.IOException;
+import java.time.DateTimeException;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Locale;
+
+public class BandcampExtractorHelper {
+
+    /**
+     * <p>Get an attribute of a web page as JSON
+     *
+     * <p>Originally a part of bandcampDirect.</p>
+     *
+     * @param html     The HTML where the JSON we're looking for is stored inside a
+     *                 variable inside some JavaScript block
+     * @param variable Name of the variable
+     * @return The JsonObject stored in the variable with this name
+     */
+    public static JsonObject getJsonData(final String html, final String variable)
+            throws JsonParserException, ArrayIndexOutOfBoundsException {
+        final Document document = Jsoup.parse(html);
+        final String json = document.getElementsByAttribute(variable).attr(variable);
+        return JsonParser.object().from(json);
+    }
```

Comment:
This is not specific to bandcamp is it? Meaning other services could theoretically use it and it should go to `ExtractorHelper`

Reply:
I'd move this to `JsonUtils`

Reply:
I'm not sure how well this would transfer to other websites… I don't know whether this is a common way of giving the JavaScript some data, but there are certainly many others as well. Bandcamp also used to do it differently.

Reply:
Moved. In case Bandcamp changes something, we can have a look again. But it is not bad to have such a method in our Utils.